api/src/test/java/uk/gov/dhsc/htbhf/claimant/creator/ClaimantLoader.java

```diff
@@ -154,7 +153,7 @@ private PaymentCycle createPaymentCycleEndingYesterday(Claim claim, List<ChildIn
                 .voucherEntitlement(voucherEntitlement)
                 .eligibilityStatus(EligibilityStatus.ELIGIBLE)
                 .identityAndEligibilityResponse(
-                        anIdentityMatchedEligibilityConfirmedUCResponseWithAllMatches(VerificationOutcome.NOT_SUPPLIED, childrenDatesOfBirth))
+                        anIdentityMatchedEligibilityConfirmedUCResponseWithAllMatches(childrenDatesOfBirth))
```

Comment:
what happened to the VerificationOutcome - is NOT_SUPPLIED the default?

Reply:
Yes, that was done in the previous java-common PR, almost all usages were changing it to NOT_SUPPLIED so it seemed sensible to make it the default in the factory.